sru/csrc/main_test_cpp.cpp

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
can we consider moving this file into a folder with a name like `tests`, to keep it separate from the actual sru code please?

Reply:
I can do that. I need your help on writing the `CMakeLists.txt` though. Need to add_library(..) from a different directory.
any idea?

Reply:
sure. Can you provide a README.md on how to run cmake please? If I run it from csrs, I get an error about findtorch.cmake not found:

```
(base) ubuntu@hughohp2:~/git/sru/sru/csrc$ cmake .
...
CMake Error at CMakeLists.txt:4 (find_package):
  By not providing "FindTorch.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Torch", but
  CMake did not find one.

  Could not find a package configuration file provided by "Torch" with any of
  the following names:

    TorchConfig.cmake
    torch-config.cmake

  Add the installation prefix of "Torch" to CMAKE_PREFIX_PATH or set
  "Torch_DIR" to a directory containing one of the above files.  If "Torch"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!
See also "/persist/git/sru/sru/csrc/CMakeFiles/CMakeOutput.log".
```

Reply:
Ok, I set TORCH_LIBRARY to `/Users/nycloaner03/conda/lib/python3.7/site-packages/torch/share/cmake/Torch`, and that configured and generated ok.